reflectometry/amor_reduction.ipynb

```diff
@@ -0,0 +1,306 @@
+    "Again, detector and wavelength masking is performed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "proud-silly",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "reference = AmorReference(reference_file)\n",
+    "reference.detector_masking(y_min=0 * sc.units.m, y_max=100e-3 * sc.units.m)\n",
+    "reference.wavelength_masking()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "substantial-doubt",
+   "metadata": {},
+   "source": [
+    "For the normalisation of the sample, we use the `Normalisation` class."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "close-mouth",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "norm = Normalisation(sample, reference)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "broke-olympus",
+   "metadata": {},
+   "source": [
+    "With this object, there is the choice to bin in the $\\lambda$/$\\theta$-space or the $q_z$-space. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "textile-moscow",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bins = (np.linspace(2.5, 15, 50), np.linspace(0.6, 1.25, 50))\n",
+    "lambda_theta = norm.wavelength_theta_bin(bins)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ambient-string",
+   "metadata": {},
+   "source": [
+    "The data binned in $\\lambda$/$\\theta$-space can be investigated and plotted."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "above-contractor",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "lambda_theta"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "hollow-practitioner",
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "sc.plot(lambda_theta, norm='log')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "knowing-ukraine",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "q_bins = np.linspace(0.007, 0.09, 200)\n",
+    "q = norm.q_bin(q_bins)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "stuck-return",
+   "metadata": {},
+   "source": [
+    "As can be done for the $q_z$-binned data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "closing-recording",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig = sc.plot(q, norm='log')\n",
+    "fig.ax.set_ylim((1e-3, 10))\n",
+    "fig"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "seeing-block",
+   "metadata": {},
+   "source": [
+    "This data can be written to an [ORSO](https://reflectometry.org)-compatible data file. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "advanced-dragon",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "norm.write_reflectometry('output.ort', bin_kwargs={'bins': q_bins})"
```

Comment:
A file is being written here, and I see it is included below. Do we need as part of the git repo?